aoc/src/main/scala/aoc2015/Day03.scala

```diff
@@ -0,0 +1,60 @@
+package aoc2015
+
+import scala.io.Source
+import scala.collection.mutable.ListBuffer
+
+object Day03 extends App:
+  val lines: String =
+    Source
+      .fromFile("aoc/src/resources/aoc2015/Day03")
+      .getLines
+      .mkString
+
+  def generateDirections(): Int =
+    var evenDirections: String = ""
+    var unevenDirections: String = ""
+    lines.zipWithIndex.foreach {
+      case (x, i) =>
```

Comment:
syntax preference on: if statements as part of the match cases; and the `+` operator for string concatenation, ie.

```
lines.zipWithIndex.foreach {
  case (x,i) if (i % 2 == 0) => evenDirections = evenDirections + x.toString
  case (x,i)                 => unevenDirections = unevenDirections + x.toString
```